### PR TITLE
Fix nutanix pull secret issue

### PR DESCRIPTION
**Pull request by sgaoshang**

### CODE REVIEW

**Comment by sgaoshang** [comment]
@jianlinliu pls help to review it, thanks.

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -4890,6 +4890,39 @@ secret_configs:
...
+      - auth_field: auth
+        email_field: email
+        item: registry.connect.redhat.com-pull-secret
+        registry_url: registry.connect.redhat.com
+      - auth_field: auth
+        email_field: email
+        item: registry.redhat.io-pull-secret
+        registry_url: registry.redhat.io
+  to:
+  - cluster_groups:
```
Comment (jianlinliu) [inline core-services/ci-secret-bootstrap/_config.yaml:4922]: I think we can append this to https://github.com/openshift/release/blob/bfa728fb645d5473dbcf84f3d3343ab6b7cb4b7d/core-services/ci-secret-bootstrap/_config.yaml#L4855 together with nutanix-qe-dis like:
```
  to:
  - cluster_groups:
    - non_app_ci
    name: cluster-secrets-nutanix-qe
    namespace: ci
  - cluster_groups:
    - non_app_ci
    name: cluster-secrets-nutanix-qe-zone
    namespace: ci
  - cluster_groups:
    - non_app_ci
    name: cluster-secrets-nutanix-qe-zone
    namespace: ci
```

Comment (sgaoshang) [reply core-services/ci-secret-bootstrap/_config.yaml:4864]: Fixed, pls review again, thanks.

**Comment by sgaoshang** [comment]
@bear-redhat Could you help to approve it? Thanks.

**Pull request merged**